src/System.Runtime/tests/System/CharTests.netcoreapp.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text.Unicode.Tests;
+using Xunit;
+using Xunit.Sdk;
+
+namespace System.Tests
+{
+    public partial class CharTests
+    {
+        [OuterLoop]
+        [Fact]
+        public static void GetUnicodeCategory_Char_AllInputs()
+        {
+            // This tests calls char.GetUnicodeCategory for every possible input, ensuring that
+            // the runtime agrees with the data in the core Unicode files. We start with U+0100
+            // because the char type special-cases values in the Latin-1 range.
```

Comment:
> because the char type special-cases values in the Latin-1 range

Does that mean the test would fail if you started at 0 instead of 0x100?

Reply:
Correct. For example, consider the character '§' (U+00A7 SECTION SIGN). `char.GetUnicodeCategory` returns `OtherSymbol`, but `CharUnicodeInfo.GetUnicodeCategory` returns `OtherPunctuation`. The `CharUnicodeInfo` class uses the latest Unicode data files, and the `char` class (for values `0000..00FF`) is locked to a much earlier version of the standard. Check the _Remarks_ section at https://docs.microsoft.com/en-us/dotnet/api/system.char.getunicodecategory for more info.

Reply:
Updated the test to start from 0, special-casing the changed code points.